packages/zoe/src/contracts/callSpread/pricedCallSpread.js

```diff
@@ -154,14 +157,9 @@ const start = zcf => {
     });
   }
 
-  // TODO(2282): change the API so the caller can provide share in basis points
-  //  rather than percent
-  function makeInvitationPair(longCollateralShare) {
-    const longPercent = makeRatio(
```

Comment:
@Chris-Hibbert am I on the right track here 🤔 or am I missing it?

```js
const longValue = longcollateralShare <= 10000 ? 
longcollateralShare 
: 
makeRatio(longCollateralShare * BASIS_POINTS) / PERCENT_BASE,
      brands.Collateral,
      BASIS_POINTS,
    );

 const longInvitation = makeOptionInvitation(Position.LONG, longValue);
```

Reply:
Oops!  I entered this comment a while ago, but apparently never submitted. Sorry about that.